src/Symfony/Component/Routing/Tests/Fixtures/dumper/url_matcher2.php

```diff
@@ -35,8 +42,8 @@ public function match($pathinfo)
         if (0 === strpos($pathinfo, '/bar')) {
             // bar
             if (preg_match('#^/bar/(?P<foo>[^/]++)$#s', $pathinfo, $matches)) {
-                if (!in_array($this->context->getMethod(), array('GET', 'HEAD'))) {
-                    $allow = array_merge($allow, array('GET', 'HEAD'));
+                if (!in_array($isLikeGetMethod, array('GET'))) {
+                    $allow = array_merge($allow, array('GET'));
```

Comment:
@stof I'm open to suggestions here. The name was given by @fabpot initially, I couldn't come up with a better name for it now. The name goes into the realm of "normalised" pretty quickly which doesn't add a lot of context.